packages/www/lib/utils/index.tsx

```diff
@@ -145,6 +145,10 @@ export function blocksToText(blocks, opts = {}) {
  */
 let stripePromise: Promise<Stripe | null>;
 export const getStripe = () => {
+  const key = process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY;
+  if (!key) {
+    return null;
```

Comment:
**nit:** I think this should be OK for JS craziness and wont cause a NRE on someone `await`ing on a null promise, but WDYT of also returning a promise here?
```suggestion
    return Promise.resolve(null);
```